R/root.R

```diff
@@ -24,6 +24,16 @@
 ##' path <- withr::local_tempfile()
 ##' hipercow_init(path)
 hipercow_init <- function(root = ".", driver = NULL, ...) {
+  if ((is.null(driver)) && (length(list(...)) > 0)) {
```

Comment:
Perhaps look into `rlang::check_dots_used` (https://rlang.r-lib.org/reference/check_dots_used.html) which might do this for us?

Reply:
There is the possibility of... 

if (is.null(driver)) {
  rlang::check_dots_empty()
}

but converting this from an error to a warning, and including a message to say why 

! `...` must be empty.

only applies when driver = NULL was a bit fiddly.